testing/btest/Baseline/core.tunnels.gre-over-udp/conn.log

```diff
@@ -0,0 +1,14 @@
+### BTest baseline data generated by btest-diff. Do not edit. Use "btest -U/-u" to update. Requires BTest >= 0.63.
+#separator \x09
+#set_separator	,
+#empty_field	(empty)
+#unset_field	-
+#path	conn
+#open XXXX-XX-XX-XX-XX-XX
+#fields	ts	uid	id.orig_h	id.orig_p	id.resp_h	id.resp_p	proto	service	duration	orig_bytes	resp_bytes	conn_state	local_orig	local_resp	missed_bytes	history	orig_pkts	orig_ip_bytes	resp_pkts	resp_ip_bytes	tunnel_parents
+#types	time	string	addr	port	addr	port	enum	string	interval	count	count	string	bool	bool	count	string	count	count	count	count	set[string]
+XXXXXXXXXX.XXXXXX	C4J4Th3PJpwUYZZ6gc	172.17.0.5	45690	1.1.1.1	53	udp	dns	0.000158	52	0	S0	T	F	0	D	2	108	0	0	ClEkJM2Vm5giqnMf4h
+XXXXXXXXXX.XXXXXX	CUM0KZ3MLUfNB0cl11	172.17.0.5	47478	192.0.78.150	80	tcp	http	0.090287	72	0	SH	T	F	0	SADF	6	332	0	0	ClEkJM2Vm5giqnMf4h
```

Comment:
Could try running the test with `-C`  and see if  `SADF` looks better. I didn't pay too much attention recording the pcap and it was mostly virtual interfaces, so not surprised checksums dont work.

Reply:
Zeek doesn't seem to care about the checksums actually. A normal run of zeek against the pcap doesn't report anything, and the btest doesn't report any difference when running with `-C`.

Reply:
Hmm, the pcap really contains just one side of the connection, so that must have been a fail on my side recording the pcap. Let me check if I can do a better one.